### DISCUSSION TITLE
Review - Flex Call Transfers to External Numbers

### DISCUSSION THREAD

**Discussion by nicedisguise** [twilio-labs/paste#967] [Office hours] [answered]
**Background:** We want to provide customers with the ability to make call transfers to external numbers (basically, meaning other departments outside of Flex). This feature only exists as a plugin currently and our efforts are to provide this out of the box for customers.
**Stage of Design:** Hi Fi Prototyping
**Links to mockups or prototype:** [Here](https://twilio.invisionapp.com/share/QEYOPVNCRAJ#/430892546_External_Call_Transfers_Cover)

**Comment by richbachman** [comment]
Hi @nicedisguise 

Thanks for posting this review topic. Do you have specific questions or feedback you are looking for from the team? Thanks.

**Reply by nicedisguise** [reply]
Hey @richbachman. 
This feature hasn't been reviewed by the team yet so I'd love a fresh perspective and to catch any issues before we start build in the new year.
With this feature we are somewhat tied to the existing UI for transfers (and Flex's UI generally) but I'm very keen to get feedback from the team for future iterations. For context: we are planning to update the UI in 2021 to UDL and would love to be able to add an updated version of the UI that reflects this to the UX Brief.

**Comment by serifluous** [comment, marked as answer]
Summary of feedback from office hours with @nicedisguise : 

- For accessibility: [Avoid showing actions on hover](https://twilio.invisionapp.com/share/QEYOPVNCRAJ#/screens/430898572). One suggestion is providing an option to click the entire row to show the action, then click to trigger or confirm the action.
- For accessibility: Consider the legibility of a screen that has multiple red buttons, especially if they're actions of different severity. The actions could be a little clearer since they're just icons, which would be even less legible at smaller sizes.
- For the Post MVP flow: [The focus state change](https://twilio.invisionapp.com/share/QEYOPVNCRAJ#/screens/430955792) could be a little confusing since the entire view of that panel changes. It might not be immediately clear what’s being filtered or searched through, and it could look like it’s just searching through the list of 9 things in view.

For transitioning to Paste, some good first steps would be to:

- Start swapping text to sentence case and remove the additional letterspacing.
- Start requesting icons used here to be added to Paste
- If the "Close dialpad" and "Back to directory" links go to new URLs, they're good as is. Otherwise, they'll have to be buttons. Off the top of my head, I can't think of any pattern that's too similar in the other products, so it might be good to ask around the PDs to see if there's something you could try to align to.

**Reply by nicedisguise** [reply]
Thanks all for your feedback!
